AirCasting/SDSync/ViewsFlow/ClearingSDCard/ClearingSDCardView.swift

```diff
@@ -28,10 +30,17 @@ struct ClearingSDCardView<VM: ClearingSDCardViewModel>: View {
             }
             Spacer()
         }
+        .alert(isPresented: $showAlert) {
+            alert
+        }
         .onAppear(perform: {
-            DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(10)) {
-                viewModel.presentNextScreen.toggle()
-            }
+            viewModel.clearSDCard()
+        })
+        .onReceive(viewModel.isClearingCompleted, perform: { result in
+            viewModel.presentNextScreen = result
+        })
```

Comment:
this `onReceive` is not needed. It is subscribed to viewmodel only to switch execution back into viewmodel. So it can be fully contained inside viewmodel implementation instead.